docs/_includes/confluence.adoc

```diff
@@ -0,0 +1,70 @@
+////
+Header: Convert Confluence XHTML to Asciidoctor
+
+Included in:
+
+- user-manual
+////
+
+You can convert Atlassian Confluence XHTML pages to Asciidoctor using this http://www.groovy-lang.org/download.html[Groovy] script.
+
+The script calls the http://pandoc.org/[Pandoc] tool to convert single or multiple HTML files exported from Confluence to AsciiDoc files (using the Python version of the AsciiDoc toolchain).
```

Comment:
Let me know if you're going to make this update or you need me to do it. Don't worry about amending or rebasing at this point. Just push the change and I'll make any adjustments necessary to the commits.